docs/release-notes/experiments-failed-status.rst

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Bug Fixes**
+
+-  Experiments: Report an experiment's status as FAILED if there is failed behavior while
+   experiments are shutting down, but have not gracefully completed.
```

Comment:
I don't understand how this might happen. Does it happen if an experiment completes but does not shut down gracefully?

Reply:
For example, a trial's container is deleted (and thus fails) while an experiment is shutting down gracefully. The failed behavior of the trial should trump the experiment's graceful shutdown.